exercises/word-count/word-count.spec.js

```diff
@@ -29,12 +29,12 @@ describe('count()', function() {
   });
 
   xit('respects case', function() {
```

Comment:
Yes, you are right. Do you mind changing it? Thanks!
On Mon, Apr 4, 2016 at 3:54 PM Rubén Chavarría <notifications@github.com>
wrote:

> In exercises/word-count/word-count.spec.js
> <https://github.com/exercism/xjavascript/pull/179#discussion_r58437061>:
>
> > @@ -29,12 +29,12 @@ describe('count()', function() {
> >    });
> >
> >    xit('respects case', function() {
>
> I think this description does not reflect what the spec does now. What
> about *normalizes to lower case*? @matthewmorgan
> <https://github.com/matthewmorgan>, if you agree, I can do the change
> while merging the PR.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/exercism/xjavascript/pull/179/files/27c5e09534fecf900a9edaf03c10a72a5b662058#r58437061>
>